.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Description
+
+Thanks for making a contribution to Cesium! We appreciate it. If this is your first PR, please see the more thorough contributing guides here:
+https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines
+
+Please include a brief summary of your changes and any context, motivation or justification you'd like the reviewer to know. You can also use this space to describe any testing you've done and how you know your code works.
+
+### Related Issues
+
+Put links to all issue(s) this PR resolves, and/or any related forum threads. 
+
+### Checklist 
+
+_Mark completed items with by placing an `x` between the square brackets. If an item is not relevant, ~strike~ it instead of marking as complete._
+
+- [ ] I've added public API changes to `CHANGES.md`.
+- [ ] I've run all the tests and confirmed that all are passing.
+- [ ] I've added tests for new code I wrote and confirmed 100% test coverage (see [testing guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/TestingGuide/README.md#running-the-tests) ).
+- [ ] I've run `npm run eslint` to confirm there are no style errors.
+- [ ] Confirmed that it runs in all browsers (by running `npm run test -- --browsers Firefox,Chrome,Edge,Safari`).
```

Comment:
I think we should be testing this visually to confirm, not just running all the command link tests.

Reply:
I'm not sure what you mean by `visually`, but I don't think it applies to the unit tests.  Certainly any new bugfix/feature should be tested, and often there will be a Sandcastle example or reproducible bug report, but running the tests themselves on the command line is ideal.

I'm not sure we can recommend anything other than Chrome to start though, since Cesium has failures in all of the others (which is something we need to fix but is a separate issue)

Reply:
I think the idea of this checkbox is "make sure Cesium doesn't insta-crash when you open the new feature in IE"

Reply:
OK, that makes sense.  Then we definitely should say via Cesium Viewer instead of unit tests.

Reply:
So what I'm hearing is we want to encourage (1) run all the unit tests on at least Chrome and (2) try running the Cesium viewer in all browsers to make sure at least the hello world/sandcastle still works?